### PR TITLE
Add media query on desktop and smaller mobile screen

### PR DESCRIPTION
**Pull request by koachcode**
# Desktop Version ( view Screen )
1. Add Desktop view
1. Add screen view for smaller mobile screen
1. Add Smooth Scroll effect

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -541,3 +557,288 @@ textarea:focus {
...
+  }
+
+  .project-img {
+    height: 100%;
+  }
+
+  .project-show-case {
+    width: 95%;
+    margin: 0 auto;
+    margin-top: 114px;
```
Comment (keneogu) [inline style.css]: let's fix this there's too much top margin from the project-show-case section please reduce it to fit the actual design.
your design
mar

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @Ginohmk, 

Good job so far!

- Linters work properly
- Descriptive Pull request title and summary
- Good design

There are some issues that you still need to work on to go to the next project but you are almost there! Yeah... you're almost there.



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: style.css (hunk 1 above)
Comment (koachcode) [reply style.css]: I have resolved the issue. thanks for pointing it out.

**Comment by koachcode** [comment]
I have amended the changes, please what am I to do now. wait for approval
or merge?

**Review by andersonlebon — APPROVED** [review]
## Hello @Ginohmk 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub flow :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your Design is looking good :clap:  :heavy_check_mark: 

## Optional suggestions
- [x] KIndly remove the ***HELLO WORLD*** headings in your README file to make it more professional. :clap: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by koachcode** [comment]
Thanks for the feedback!  and Thanks for the tip on making my Readme
professional

**Pull request merged**